src/data/data.js

```diff
@@ -78,6 +78,47 @@ const pageData = {
           },
         ],
       },
+      {
+        component: 'footer',
+        logo: {
+          href: 'http://www.google.com',
+          image: {
+            src: 'footer-logo.png',
+            alt: 'SER HABIL',
+          },
+        },
+        items: [
+          {
+            text: 'Eventos',
+            href: 'http://www.google.com',
+          },
+          {
+            text: 'Fundacion',
```

Comment:
Fundación needs the accent on the letter o

Reply:
<img width="241" alt="Screen Shot 2020-08-02 at 11 04 45 AM" src="https://user-images.githubusercontent.com/54866521/89127068-51cddd00-d4b0-11ea-8752-144c3e968f92.png">


Fixed the missing foundation accent